docs/docs/features/persist-state.mdx

```diff
@@ -80,3 +80,19 @@ export const persist = persistState(authStore, {
   storage: localForage,
 });
 ```
+
+## Excluding keys from saved state
+
+`excludeKeys()` operator can be used to exclude some keys from saved state:
+
+```ts
+import { excludeKeys, persistState, localStorageStrategy } from '@ngneat/elf-persist-state';
+
+persistState(todoStore, {
+  key: 'todos',
+  storage: localStorageStrategy,
+  source: () => todoStore.pipe(
+    excludeKeys(['ids', 'entities'])
```

Comment:
I think that `omitKeys` is more suitable. See lodash naming for example https://lodash.com/docs/4.17.15#omit

Reply:
Just a matter of personal preference. `omit` might be better because it's shorter and sounds a bit more formal, but `exclude` is more widely used word and when you have auto-completion in your IDE the length doesn't really matter.
Anyway, you can choose whichever you prefer.